sycl/include/CL/sycl/INTEL/esimd/esimd_memory.hpp

```diff
@@ -643,6 +643,198 @@ media_block_store(AccessorTy acc, unsigned x, unsigned y, simd<T, m * n> vals) {
 SYCL_EXTERNAL void slm_init(uint32_t size) {}
 
 #endif
+
+template <typename AccessorTy>
+ESIMD_INLINE ESIMD_NODEBUG uint32_t esimd_get_value(AccessorTy acc) {
+#if defined(__SYCL_DEVICE_ONLY__) && defined(__SYCL_EXPLICIT_SIMD__)
+  return __esimd_get_value(AccessorPrivateProxy::getNativeImageObj(acc));
+#else
+  return __esimd_get_value(acc);
+#endif // __SYCL_DEVICE_ONLY__ && __SYCL_EXPLICIT_SIMD__
+}
+
+/// \brief Raw sends load.
+///
```

Comment:
Please provide some comments what is "Raw send".
Does it model "send" instruction from https://01.org/sites/default/files/documentation/intel-gfx-prm-osrc-icllp-vol02a-commandreference-instructions_2.pdf? Maybe it makes sense to add a link then?

Reply:
Yes, this API is used to implement the send messages. Added link to the open source document.